db/migrate/20221206080208_create_articles.rb

```diff
@@ -0,0 +1,11 @@
+class CreateArticles < ActiveRecord::Migration[7.0]
+  def change
+    create_table :articles do |t|
+      t.string :creator_name
+      t.string :article_title
+      t.text :article_body
```

Comment:
maybe we can simplify it a bit, e.g:
```suggestion
      t.string :creator
      t.string :title
      t.text :body
```